helper_test.go

```diff
@@ -343,7 +343,7 @@ func createPlannedRun(t *testing.T, client *Client, w *Workspace) (*Run, func())
 		}
 
 		switch r.Status {
-		case RunPlanned, RunCostEstimated, RunPolicyChecked, RunPolicyOverride:
```

Comment:
I think we should keep this state as in some cases it can be returned right?

Reply:
See other comment.